web/public/js/directives/cd-picture-grid/template.dust

```diff
@@ -1,7 +1,9 @@
 <div>
-  <div class="cd-picture-grid" ng-class="{'cd-picture-grid--wide-card': $ctrl.wideCard}">
-    <a class="cd-picture-grid__item" ng-repeat="item in $ctrl.items" ng-href="{{ item.href }}" ng-click="$ctrl.handleSelection($event, item)">
-      <div class="cd-picture-grid__picture" style="background-image: url({{ item.picture }});"></div>
+  <div class="cd-picture-grid" ng-class="{'cd-picture-grid--wide-card': $ctrl.wideCard, 'cd-picture-grid--left-align': $ctrl.align === 'left'}">
```

Comment:
Nothing different unless certain parameters are passed to cd-picture-grid. It's only adding classes. I'll double check things like manage users still works, but I know use of it in view profile works fine.